src/com/google/foam/demos/sevenguis/CircleDrawer.js

```diff
@@ -156,18 +156,15 @@ foam.CLASS({
 
       if ( ! this.selected ) return;
 
-      var p = this.PopupView.create({
-        width: 450,
-        height: 110
-      }).add(this.DiameterDialog.create({data: this.selected}));
-
-      this.add(p);
+      var p = this.PopupView.create()
+          .add(this.DiameterDialog.create({ data: this.selected }));
 
       // If the size is changed with the dialog, then create an updated memento
       var oldRadius = this.selected.radius;
       p.onunload.sub(function() {
         if ( this.selected.radius !== oldRadius ) this.updateMemento();
       }.bind(this));
+      p.open();
```

Comment:
Then I submit that these are actually different views for different purposes.

I'll rename my version to DialogContainer or something.